docs/eventing/getting-started.md

```diff
@@ -0,0 +1,475 @@
+Now, you can make a HTTP request. To show the various types of events you can send, you will make three requests.
+
+1. To make the first request, run the following in the SSH terminal:
+
+    ```sh
+    curl -v "default-broker.event-example.svc.cluster.local" \
+      -X POST \
+      -H "Ce-Id: say-hello" \
+      -H "Ce-Specversion: 0.2" \
+      -H "Ce-Type: greeting" \
+      -H "Ce-Source: not-sendoff" \
+      -H "Content-Type: application/json" \
+      -d '{"msg":"Hello Knative!"}'
+    ```
+
+    This creates a `CloudEvent` that has the `type` `greeting`. When an event is sent to the `Broker`, the `Trigger` `hello-display` will activate and send it to the event consumer of the same name.
+
+    If the event has been received, you will receive a `202 Accepted` response.
+
+
+2. To make the second request, run the following in the SSH terminal:
+
+    ```sh
+    curl -v "default-broker.event-example.svc.cluster.local" \
+      -X POST \
+      -H "Ce-Id: say-goodbye" \
+      -H "Ce-Specversion: 0.2" \
+      -H "Ce-Type: not-greeting" \
+      -H "Ce-Source: sendoff" \
+      -H "Content-Type: application/json" \
+      -d '{"msg":"Goodbye Knative!"}'
+    ```
+
+    This creates a `CloudEvent` that has the `source` `sendoff`. When the event is sent to the `Broker`, the `Trigger` `goodbye-display` will activate and send it to the event consumer of the same name.
+
+    If the event has been received, you will receive a `202 Accepted` response.
+
+
+3. To make the third request, run the following in the SSH terminal:
+
+    ```sh
+    curl -v "default-broker.event-example.svc.cluster.local" \
+      -X POST \
+      -H "Ce-Id: say-hello-goodbye" \
+      -H "Ce-Specversion: 0.2" \
+      -H "Ce-Type: greeting" \
+      -H "Ce-Source: sendoff" \
+      -H "Content-Type: application/json" \
+      -d '{"msg":"Hello Knative! Goodbye Knative!"}'
+   ```
+
+  This creates a `CloudEvent` that has the `type` `greeting` and the`source` `sendoff`. When the event is sent to the `Broker`, the `Triggers` `hello-display` and `goodbye-display` will activate and send it to the event consumer of the same name.
+
+  If the event has been received, you should receive a `202 Accepted` response.
+
+4. Exit SSH.
+
+If everything has been done correctly, you should have sent 2 `CloudEvents` to the `hello-display` event consumer and 2 `CloudEvents` to the `goodbye-display` event consumer (note that `say-hello-goodbye` is sent to *both* `hello-display` and `goodbye-display`). You will verify that these events were received correctly in the next section.
+
+## Verifying events were received {:#verify}
+
+After sending events, verify that the events were received by the appropriate `Subscribers`.
+
+1. Look at the logs for the `hello-display` event consumer by running the following command:
+
+    ```sh
+    kubectl -n event-example logs -l app=hello-display
+    ```
+
+
+    The command shows the `Attributes` and `Data` of the events you sent to `hello-display`:
```

Comment:
@grantr I don't get the expected results that we documented here.

I only ever get the last event with far less detail:

```
[rescarez] ~/knative/docs (master)
$ kubectl -n event-example logs -l app=hello-display
  source: sendoff
  id: say-hello-goodbye
  time: 2019-08-12T19:49:09.7953344Z
  contenttype: application/json
Extensions,
  knativehistory: default-kn-trigger-channel-cqjls.event-example.svc.cluster.local
Data,
  {
    "msg": "Hello Knative! Goodbye Knative!"
  }

[rescarez] ~/knative/docs (master)
$ kubectl -n event-example logs -l app=goodbye-display
  source: sendoff
  id: say-hello-goodbye
  time: 2019-08-12T19:49:09.7953344Z
  contenttype: application/json
Extensions,
  knativehistory: default-kn-trigger-channel-cqjls.event-example.svc.cluster.local
Data,
  {
    "msg": "Hello Knative! Goodbye Knative!"
  }

```

So if I rerun the first two curl commands, i see:

```
[rescarez] ~/knative/docs (master)
$ kubectl -n event-example logs -l app=hello-display
  source: not-sendoff
  id: say-hello
  time: 2019-08-12T20:25:46.104264737Z
  contenttype: application/json
Extensions,
  knativehistory: default-kn-trigger-channel-cqjls.event-example.svc.cluster.local
Data,
  {
    "msg": "Hello Knative!"
  }

[rescarez] ~/knative/docs (master)
$ kubectl -n event-example logs -l app=goodbye-display
  source: sendoff
  id: say-goodbye
  time: 2019-08-12T20:25:12.895347894Z
  contenttype: application/json
Extensions,
  knativehistory: default-kn-trigger-channel-cqjls.event-example.svc.cluster.local
Data,
  {
    "msg": "Goodbye Knative!"
  }

```

Reply:
@RichieEscarez `kubectl logs` when used with a selector (`-l app=goodbye-display`) only shows the 10 most recent log lines. You can change this with the `--tail=N` option which specifies the number of lines to display. Alternately `--tail=-1` displays all lines.

```
kubectl logs -l app=goodbye-display --tail=100
```

Reply:
Thanks Grant! I'll use `--tail=-1` just to cover everything.